ln-simln-jamming/src/main.rs

```diff
@@ -182,7 +183,8 @@ async fn main() -> Result<(), BoxError> {
         clock.clone(),
         &network_dir.sim_network,
         target_pubkey,
-        attacker_pubkey,
+        // For sink attack we only use one attacker node so just take the first one.
+        attacker_pubkeys[0],
```

Comment:
nit: pass the full vec in and have `new` assert that there's only one?

Happy to put this off till the "choose attack" PR since this one is ready to roll.